.github/workflows/cli.yml

```diff
@@ -17,6 +17,11 @@ on:
       - cli/go.mod
       - .github/workflows/**
 
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
```

Comment:
Remove comment

Reply:
Comments removed for everything but tests.